go/vt/vtorc/logic/tablet_discovery_test.go

```diff
@@ -277,6 +277,115 @@ func TestShardPrimary(t *testing.T) {
 	}
 }
 
+func TestGetKeyspaceShardsToWatch(t *testing.T) {
```

Comment:
@pbibra can we add a test case for `keyspace/-`, which is confusingly a single-shard. `keyspace/0` is also a single shard but I don't think this logic cares about it